### PR TITLE
Feature/iii 3495 change production name

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- Possibility to change the name of a production

Screenshot 2021-12-09 at 11 15 21

Screenshot 2021-12-09 at 11 19 39

**Error state:**
Screenshot 2021-12-10 at 12 26 45

---

Ticket: https://jira.uitdatabank.be/browse/III-3495

### CODE REVIEW

File: src/components/pages/manage/productions/index/index.js [hunk 1]
```diff
@@ -26,6 +27,8 @@ import { Productions } from './Productions';
 
 const productionsPerPage = 15;
 
+const productionsQueryKey = ['productions', { limit: 15, name: '', start: 0 }];
```
Comment (simon-debruijn) [inline src/components/pages/manage/productions/index/index.js:30]: @brampauwelyn have you tried the key 'productions' after you fixed it? I think it should work

Comment (brampauwelyn) [reply src/components/pages/manage/productions/index/index.js:30]: Nope I didn't :p 
can you try it?

Comment (simon-debruijn) [reply src/components/pages/manage/productions/index/index.js:30]: I tried it quickly, seems like you have to pass the whole key

File: src/components/pages/manage/productions/index/Events.js [hunk 2]
```diff
@@ -256,15 +329,33 @@ const Events = ({
...
+
+        {!isAddActionVisible && !isChangeNameActionVisible ? (
           <Actions
             loading={loading}
             activeProductionName={activeProductionName}
             onClickAdd={onClickAdd}
+            onClickChangeName={onClickChangeName}
             onClickDelete={onClickDelete}
             shouldDisableDeleteButton={shouldDisableDeleteButton}
           />
```
Comment (simon-debruijn) [inline src/components/pages/manage/productions/index/Events.js:357]:
```suggestion
        ) : !isAddActionVisible ? (
          <Actions
            loading={loading}
            activeProductionName={activeProductionName}
            onClickAdd={onClickAdd}
            onClickChangeName={onClickChangeName}
            onClickDelete={onClickDelete}
            shouldDisableDeleteButton={shouldDisableDeleteButton}
          /> : null}
```

You can also chain multiple conditional renders behind eachother

**Review by simon-debruijn — APPROVED** [review]

**Pull request merged**